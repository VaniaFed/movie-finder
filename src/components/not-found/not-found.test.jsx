import renderer from 'react-test-renderer';
import { NotFound } from './index';

describe('<NotFound />', () => {
    const tree = renderer
        .create(<NotFound text="Not found we're so sorry..." />)
        .toJSON();
    it('should be rendered with text "Not found we\re so sorry..."', () => {
        expect(tree).toMatchSnapshot();
    });
});
