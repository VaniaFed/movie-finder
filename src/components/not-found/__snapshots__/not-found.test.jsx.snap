// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should be rendered with text "Not found we
e so sorry..." 1`] = `
<h2
  className="not-found"
/>
`;

exports[`<NotFound /> should be rendered with text "Not found wee so sorry..." 1`] = `
<h2
  className="not-found"
/>
`;
